### PR TITLE
Add test for AI template output

### PR DESCRIPTION
**Pull request by wejendorp**
- **test: add ai-mode integration test**
- **test: add AI template test files**

### CODE REVIEW

**Review by andertun — COMMENTED** [review]
Shouldn't there be an `ai.kapeta.yml` file in `test/resources/examples`?

**Comment by wejendorp** [comment]
> Shouldn't there be an `ai.kapeta.yml` file in `test/resources/examples`?

We could, I just wanted to make sure that it works in AI mode. I suppose it'd be cleaner by having its own file.

File: test/specs/integration.test.ts [hunk 1]
```diff
@@ -31,6 +31,16 @@ describe('blocks', () => {
         return CodegenHelpers.testCodeGenFor(target, new BlockCodeGenerator(data), basedir);
     });
 
+    test.only('ai', async () => {
```
Comment (andertun) [inline test/specs/integration.test.ts:34]:
```suggestion
    test('ai', async () => {
```

**Review by andertun — APPROVED** [review]

File: test/specs/integration.test.ts (hunk 1 above)
Comment (wejendorp) [reply test/specs/integration.test.ts:34]: :o thanks! big oops

**Pull request merged**